engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -145,6 +192,55 @@ public GroovyDeephavenSession(
         }
     }
 
+    /**
+     * Adds the default imports that Groovy users assume to be present.
+     */
+    private void addDefaultImports(ImportCustomizer imports) {
+        // TODO (core#230): Remove large list of manual text-based consoleImports
+        // NOTE: Don't add to this list without a compelling reason!!! Use the user script import if possible.
+        imports.addImports(
```

Comment:
Reviewer note: I made this a list of class literals to assist in future refactoring, but can revert to just putting a list of strings for each of these.

Reply:
You could introduce a helper method to invoke getName automatically, but this doesn't look like it will be gross to maintain as is =P.